lib/Extension/ExpressionExtension.php

```diff
@@ -195,7 +197,7 @@ public function load(Container $container): void
         $container->register(HighlightingNodePrinter::class, function (Container $container) {
             return new HighlightingNodePrinter(
                 $container->get(NodePrinters::class),
-                new Standard8ColorMap()
+                class_exists(NullOutputFormatter::class) ? new TrueColorMap() : new Standard8ColorMap()
```

Comment:
Symfony Console only supports true color in 5.0: This is a class only exists in 5.0 - another option would be to use PackageVersions